passwords-evolved.php

```diff
@@ -22,7 +22,7 @@
 
 // PHP version check to check if we have to use the password_compat library.
 if (version_compare(PHP_VERSION, '5.3.7', '<')) {
-    exit(sprintf('Passwords Evolved requires PHP 5.3.7 or higher. Your WordPress site is using PHP %s.', PHP_VERSION));
+    exit(sprintf(__('Passwords Evolved requires PHP 5.3.7 or higher. Your WordPress site is using PHP %s.', 'passwords-evolved'), PHP_VERSION));
```

Comment:
Like I said in the previous PR, I don't think we can translate this. So I would take it out.